src/util.h

```diff
@@ -124,6 +124,16 @@ namespace palace
     return attributes;
   }
 
+  inline alure::FilterParams
+  make_filter_params (std::vector<float> params) noexcept
+  {
+    alure::FilterParams filter_params;
+    filter_params.mGain = params[0];
+    filter_params.mGainHF = params[1];
+    filter_params.mGainLF = params[2];
+    return filter_params;
```

Comment:
Is it possible to `return alure::FilterParams {params[0], params[1], params[2]};`?